src/toolchain/rust.rs

```diff
@@ -409,6 +409,22 @@ fn install_rust_nightly(version: &str) -> Result<()> {
     Ok(())
 }
 
+/// Unnstalls the RiscV target.
+pub fn uninstall_riscv_target(nightly_version: &str) -> Result<()> {
+    info!("{} Installing Riscv target", emoji::WRENCH);
+    cmd!(
+        "rustup",
+        "target",
+        "remove",
+        "--toolchain",
+        nightly_version,
+        "riscv32imac-unknown-none-elf"
```

Comment:
Are we _only_ installing the `imac` target here? Generally we target `imc`, as `imac` requires emulating atomics which has a performance hit, however both _are_ supported.

Reply:
Just updated it to install/uninstall both `imac` and `imc` targets.